circle.yml

```diff
@@ -1,72 +0,0 @@
-# These environment variables must be set in CircleCI UI
```

Comment:
This comment still applies, right? I still see the same env vars in use.

Reply:
Yes, I didn't change any variables.